### PR TITLE
IA-1623: run cypress tests manually

### PR DESCRIPTION
**Pull request by codeKameleon**
Explain what problem this PR is resolving

The workflow only handled cypress tests on a PR by commenting with @cypress. It didn't allow to launch a workflow manually on a specific branch

## Self proof reading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests
## Changes

- updated the workflow file to add an event trigger : `workflow_dispatch` allowing to launch cypress tests manually on a specific branch.

### Launch the cypress tests on a PR with comment
Add a comment with this body : "@cypress" on the PR you want to test

### Launch the cypress tests on a branch manually 
Go to [actions](https://github.com/BLSQ/iaso/actions) tab > select the workflow "Iaso Cypress manual testing"  > Run workflow (choose the branch you want to run the workflow)

## Print screen / video

comment

workflow_dispatch

### CODE REVIEW

**Comment by codeKameleon** [comment]
@cypress

**Comment by codeKameleon** [comment]
@cypress

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -142,16 +142,16 @@ jobs:
...
 
+
       - name: Mark comment as success
-        if: github.event.issue.pull_request && contains(github.event.comment.body, '@cypress') == true
         uses: dkershner6/reaction-action@v1
         if: success()
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           reaction: 'hooray'
+        if: github.event.issue.pull_request && contains(github.event.comment.body, '@cypress') == true
```
Comment (qgerome) [inline .github/workflows/cypress.yml]: There is already a `if` just above. You need to merge them.

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -164,4 +164,5 @@ jobs:
               repo,
               body: 'Test failed: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
             });
+        if: github.event.issue.pull_request && contains(github.event.comment.body, '@cypress') == true
```
Comment (qgerome) [inline .github/workflows/cypress.yml]: Same thing

**Review by qgerome — CHANGES_REQUESTED** [review]

**Comment by codeKameleon** [comment]
@cypress

**Comment by codeKameleon** [comment]
@cypress

**Review by qgerome — APPROVED** [review]

**Pull request merged**